src/w3c/templates/sotd.js

```diff
@@ -29,6 +28,35 @@ export default (conf, opts) => {
           </details>
         `
       : ""}
+    ${conf.isUnofficial
```

Comment:
Not sure why this change is here?

Reply:
Looking at the commit history, I think this is accidentally added when merging conflicts.

Reply:
Should I remove this? As this was a part of the merge which I did earlier

Reply:
This part had been existed, modified after you opened this PR, and then deleted, which is why this appeared when merging. Should be removed anyway.

Reply:
Ohk I'll remove it